pandas/tests/indexing/test_partial.py

```diff
@@ -637,3 +637,46 @@ def test_partial_set_empty_frame_empty_consistencies(self):
         df.loc[0, 'x'] = 1
         expected = DataFrame(dict(x=[1], y=[np.nan]))
         tm.assert_frame_equal(df, expected, check_dtype=False)
+
```

Comment:
parametrize both of these tests with strings & with Timestamp for the indexers

Reply:
either I do not understand you correctly or re-read the tests. I already both use string labels and `pd.Timestamp` labels there.